ranger2/aws/ranger_asg/ranger_policy.py

```diff
@@ -0,0 +1,161 @@
+import requests
+import json, sys
+
+#This script takes 4 arguments.
+#Loadbalancer Endpoint.
+#Defloc Location.
+#Service Name to be Created.
+#Loadbalancer Port
+#Usage: python ranger_policy.py LB-IP pritish-qubole hive_policy 80
+
+host = sys.argv[1]
+defloc_loc =  sys.argv[2] 
+service_name = sys.argv[3]
+port = sys.argv[4]
+
+#host = "ps-cvs-ranger-asg-alb-1733501465.us-east-1.elb.amazonaws.com"
+#defloc_loc =  "pritish-qubole" 
```

Comment:
we can have descriptive comments on where to get this information from.

Reply:
Fixed and added comments.